File: 0075-sort-colors/counting-sort/python/solution.py
from collections import Counter

class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        freq = collections.Counter(nums)
        idx = 0
        for color in range(3):
            while(freq[color]):
                nums[idx] = color
                freq[color] -= 1
                idx += 1

File: 0075-sort-colors/counting-sort/cpp/solution.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int freq[3] = {0, 0, 0};
        for (int num : nums) {
            freq[num]++;
        }

        int idx = 0;
        for (int color = 0; color < 3; color++) {
            while (freq[color]) {
                nums[idx] = color;
                freq[color]--;
                idx++;
            }
        }
    }
};
